hotel/src/main/java/com/andersenhotels/model/service/StateManager.java

```diff
@@ -0,0 +1,30 @@
+package com.andersenhotels.model.service;
+
+import com.andersenhotels.model.config.ConfigManager;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import lombok.Getter;
+
+import java.io.File;
+import java.io.IOException;
+
+public class StateManager {
+
+    @Getter
+    private static String PATH = ConfigManager.getStateFilePath();
+
+    public static void setPATH(String PATH) {
+        StateManager.PATH = PATH;
+    }
```

Comment:
what the idea of this PATH? its static field. not good idea to use detter here))